UnityProject/Assets/Scripts/Objects/Engineering/Reactor/ReactorGraphiteChamber.cs

```diff
@@ -165,10 +165,10 @@ public void CycleUpdate()
 				if (ReactorPipe.pipeData.mixAndVolume.Total.x > 0 &&
 					ReactorPipe.pipeData.mixAndVolume.Temperature > BoilingPoint)
 				{
-					var ExcessEnergy = (ReactorPipe.pipeData.mixAndVolume.Temperature - BoilingPoint);
-					ReactorPipe.pipeData.mixAndVolume.Remove((ReactorPipe.pipeData
-						.mixAndVolume.Total * (EnergyToEvaporateWaterPer1
-											   * ExcessEnergy * WaterEnergyDensityPer1)));
+					var ExcessEnergy = ReactorPipe.pipeData.mixAndVolume.Temperature - BoilingPoint;
+					ExcessEnergy *= EnergyToEvaporateWaterPer1 * WaterEnergyDensityPer1;
```

Comment:
just remember to be careful here since some NAN have come from Reactor and pipes